codec-http2/src/main/java/io/netty/handler/codec/http2/internal/hpack/DynamicTable.java

```diff
@@ -169,7 +169,7 @@ public void setCapacity(int capacity) {
             }
         }
 
-        int maxEntries = capacity / HEADER_ENTRY_OVERHEAD;
+        int maxEntries = (int) capacity / HEADER_ENTRY_OVERHEAD;
```

Comment:
This will first cast `capacity` to int, then do the division. I think you need to wrap the division with parantheses?